keadm/cmd/keadm/app/cmd/beta/beta.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package beta
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	keadmLongDescription = `
+    +----------------------------------------------------------+
+    | KEADM                                                    |
+    | Easily bootstrap a KubeEdge cluster                      |
+    |                                                          |
+    | Please give us feedback at:                              |
+    | https://github.com/kubeedge/kubeedge/issues              |
+    +----------------------------------------------------------+
+
+    Create a cluster with cloud node
+    (which controls the edge node cluster), and edge nodes
+    (where native containerized application, in the form of
+    pods and deployments run), connects to devices.
+
+`
+	keadmExample = `
+    +----------------------------------------------------------+
+    | On the cloud machine:                                    |
+    +----------------------------------------------------------+
+    | master node (on the cloud)# sudo keadm beta init         |
+    +----------------------------------------------------------+
+
+    +----------------------------------------------------------+
+    | On the edge machine:                                     |
+    +----------------------------------------------------------+
+    | worker node (at the edge)# sudo keadm beta join <flags>  |
+    +----------------------------------------------------------+
+
+    You can then repeat the second step on, as many other machines as you like.
```

Comment:
We have the introdunction of keadm when only running `keadm`, so this description should descripts the meaning of the `beta` sub-command itself, only a short maybe enough.
We can add more description in `beta init` and `beta join` and so on. How do you think? @zhu733756 @zc2638 @gy95

Reply:
I agree with you.
At first thought, the beta version of keadm does, so bring the full description in. 
When the subcommands under beta are migrated to the main command, beta can still be used, but it is not appropriate to describe the overall keadm.
@fisherxu